templates/frag/parseMultipartValue.hbs

```diff
@@ -0,0 +1,11 @@
+{{!--
+Determine the correct value to pass from a multipart property to the FromJsonFunction.
```

Comment:
I think "pass" should be "parse". Maybe we need the same change in the JSON equivalent? I think this file should be named `extract` rather than `parse` actually.

```
Extract the value from the multipart input.
```

Reply:
There's no JSON equivalent for this.